tests/integration/generic/read-only-hook.test.js

```diff
@@ -11,6 +11,9 @@ test(
       90
     );
 
+    // This is necessary to ensure that the HTTP-Server already logged the Request
+    await new Promise(resolve => setTimeout(resolve, 5000));
```

Comment:
How stable is this?
I've often seen tests with static wait times get janky and fail randomly.